README.md

```diff
@@ -1,16 +1,13 @@
-# Dyspatch Ruby API Client
+# dyspatch_client
```

Comment:
```suggestion
# Dyspatch Ruby API Client
```

Reply:
The only issue with this, is this file is generated, so any fixes for this should be handled as part of the readme updates in another task.

Reply:
Sounds good! Reverted back